packages/web/src/components/form2/FormFieldsValue.tsx

```diff
@@ -49,10 +49,42 @@ export default function FormFieldsValue({
   onLayoutChange,
 }: IProps) {
   const [state, setState] = useState(initialState);
+  const [height, setheight] = useState([]);
   // const [layout, setLayout] = useState({});
 
   const setInitialState = () => setState(initialState);
 
+  const countFormHeight = (forms) => {
+    let formHeights = [];
+    forms.map(async (form) => {
+      let height = 128; //default height for each form
```

Comment:
@rajdama how can each form have the same default height? How did you calculate 128?